user-docs/modules/ROOT/pages/fedora-iot-bootc-quay-example.adoc

```diff
@@ -0,0 +1,106 @@
+= Fedora IoT Bootc Image Example with Quay
+
+== Building and Booting a Fedora IoT Bootc Image
+
+This example walks through building and booting a Fedora IoT bootc image using Quay.io, as well as pushing an update to a booted 
+Fedora IoT system.
+
+To start, create a `Containerfile.custom` with the following contents:
+----
+FROM quay.io/fedora-testing/fedora-bootc:rawhide-standard as builder
+RUN /usr/libexec/bootc-base-imagectl build-rootfs --manifest=fedora-iot /target-rootfs
+
+FROM scratch
+COPY --from=builder /target-rootfs/ /
+LABEL containers.bootc 1
+ENV container=oci
+STOPSIGNAL SIGRTMIN+3
+CMD ["/sbin/init"]
+----
+
+You're now ready to build a Fedora IoT bootc image using the custom containerfile you made earlier. Use this command:
+----
+podman build --cap-add=all --security-opt=label=type:container_runtime_t \
+--device /dev/fuse -t localhost/fedora-iot -f Containerfile.custom 
```

Comment:
I think this needs the trailing . to indicate the build should happen from the current directory where the Containerfile is

```suggestion
podman build --cap-add=all --security-opt=label=type:container_runtime_t \
--device /dev/fuse -t localhost/fedora-iot -f Containerfile.custom .
```